src/MSBuild.Sdk.SqlProj/Sdk/Sdk.targets

```diff
@@ -112,15 +112,19 @@
     <ItemGroup> 
       <!-- Resolve all package references to their physical location first -->
       <_ResolvedPackageReference Include="%(PackageReference.Identity)">
-        <PhysicalLocation>$(NuGetPackageRoot)%(PackageReference.Identity)\%(PackageReference.Version)</PhysicalLocation>
+        <PhysicalLocation>$(NuGetPackageRoot)%(PackageReference.Identity)/%(PackageReference.Version)</PhysicalLocation>
       </_ResolvedPackageReference>
+      <_ResolvedDacpacFile Include="@(_ResolvedPackageReference)">
+        <DacpacFile>$([System.String]::new('%(PhysicalLocation)/tools/').ToLower())%(Identity).dacpac</DacpacFile>
+      </_ResolvedDacpacFile>
     
       <!-- Build a list of package references that include a dacpac file matching the package identity in their tools folder -->
-      <DacpacReference Include="@(_ResolvedPackageReference)" Condition="Exists('%(PhysicalLocation)\tools\%(Identity).dacpac')" />
+      <DacpacReference Include="@(_ResolvedDacpacFile)" Condition="Exists(%(DacpacFile))" />
     </ItemGroup>
-  
-    <Message Importance="normal" Text="Resolved package reference %(_ResolvedPackageReference.Identity) to %(_ResolvedPackageReference.PhysicalLocation)" />
-    <Message Importance="normal" Text="Resolved database package references: @(DacpacReference)" />
+
+    <Message Importance="High" Text="Resolved package reference %(_ResolvedPackageReference.Identity) to %(_ResolvedPackageReference.PhysicalLocation)" />
+    <Message Importance="High" Text="Resolved dacpac file from package %(_ResolvedDacpacFile.Identity) to %(_ResolvedDacpacFile.DacpacFile)" />
+    <Message Importance="High" Text="Resolved database package references: @(DacpacReference)" />
```

Comment:
As mentioned we'll probably want to lower the `Importance` back to `normal` again here.

Reply:
Done.